src/data/localization.rs

```diff
@@ -173,19 +120,33 @@ impl LocaEntry {
                 match macrovalue {
                     MacroValue::Text(ref token) => vec.push(token.clone().linked(link)),
                     MacroValue::Keyword(k, _) => {
```

Comment:
maybe replace this with `keyword` or `key`

Reply:
done